code/game/objects/items/reagent_containers/food/drinks.dm

```diff
@@ -23,6 +23,10 @@
 		to_chat(user, SPAN_DANGER("The [src.name] is empty!"))
 		return FALSE
 
+	if(HAS_TRAIT(M, TRAIT_CANNOT_EAT))
+		to_chat(user, SPAN_DANGER("[user == M ? "You are" : "[M] is"] unable to drink!"))
+		return
```

Comment:
```suggestion
		return FALSE
```

Consistency with the rest of the proc (the previous PR should have done the same but oh well)